hatchways/team-pine
# Implemented the Bookings UI

### What this PR does:
- Provides the skeleton to the Bookings page. There is a section for the bookings and a calendar we can pick dates from.
- Bookings are displayed on bottom on mobile and the left side on desktop. Calendar is on the top or the right.
- Calendar displays all upcoming non-declined bookings.

### Screenshots / Videos:
- Desktop:
![image](https://user-images.githubusercontent.com/34494229/149601115-2d9697d5-5923-42fc-a488-256d79f740e7.png)
- Mobile:
![image](https://user-images.githubusercontent.com/34494229/149601135-3d18d7cb-e62c-4a6d-9f84-c9df3f8a0da3.png)
![image](https://user-images.githubusercontent.com/34494229/149601156-1b7f3cd8-48ec-4ab3-965c-e6d68861894d.png)


### Any information needed to test this feature:
- Make sure you are logged in as any user, then navigate to /bookings.
- Click on the calendar dates and navigation features as well.

### Any issues with the current functionality:
- Bookings are mock data currently.
- We don't currently handle empty bookings. This will be easier to implement once we know the exact way we retrieve the information, which is during integration.

closes #19 

### Review thoughts:
- I wanted to use react-calendar because it had decent functionality already built in and thought it would save some time. The styling side of it was awkward, and I didn't find elegant solutions to some of the problems (like using padding for backgrounds like normal).
- I added mock information and the means to display it. The UI is much more complete now. I also feel a bit better about the calendar (the active styling is the same across all screen sizes for instance).
